pkg/manifest/manifest.go

```diff
@@ -841,14 +823,8 @@ func UnmarshalManifest(cxt *context.Context, manifestData []byte) (*Manifest, er
 		if _, found := knownFields[key]; found {
 			delete(unmappedData, key)
 		}
-		// Print deprecation notice for this field
-		if key == "invocationImage" {
-			fmt.Fprintln(cxt.Out, "WARNING: The invocationImage field has been deprecated and can no longer be user-specified; ignoring.")
```

Comment:
How about we keep this warning line? Just so that if someone is still using tag, they see something.